packages/tre/src/runtime/index.ts

```diff
@@ -21,14 +31,16 @@ export interface RuntimeConstructor {
 }
 
 const COMMON_RUNTIME_ARGS = ["serve", "--binary", "--verbose"];
```

Comment:
Unrelated to this PR, but should we expose `--verbose` as a miniflare option to control this?